### ISSUE TITLE
[EN][Dragon Soul][Warlord Zon'ozz]Warlord Zon'ozz encounter bugs 

### ISSUE THREAD

**Issue by Aival94** [Atlantiss/BugTracker#9754]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description**: 
Will describe all the current [Warlord Zon'ozz](http://www.wowhead.com/npc=55308/warlord-zonozz) Normal bugs that i find and know.

**Current behaviour**: 
1. Pets cannot reach and attack the boss.(known by devs and is mmaps bug)
2. Seems that some pets/totems can react like a player when they touch the [Void of the Unmaking](http://www.wowhead.com/npc=55334/void-of-the-unmaking)
3. It does not matter at which direction/angle you "hit" the [Void of the Unmaking](http://www.wowhead.com/npc=55334/void-of-the-unmaking) and it always follow a specific path.
4. Seems that [Void of the Unmaking](http://www.wowhead.com/npc=55334/void-of-the-unmaking) have a 3 sec cooldown before it can collide with someone again.

**Expected behaviour**: What should happen instead
1. Pets should be able to reach and attack the boss.
2. Pets/totems should not be able to colldie with [Void of the Unmaking](http://www.wowhead.com/npc=55334/void-of-the-unmaking).
3. The [Void of the Unmaking](http://www.wowhead.com/npc=55334/void-of-the-unmaking) should change its path depend of at which angle it collide with a player.
4. The [Void of the Unmaking](http://www.wowhead.com/npc=55334/void-of-the-unmaking) should probably have less inner cooldown(1.5sec) to again be able to collide with a player.

**Steps to reproduce the problem**:

1. Have pet/guardians classes in raid.
2. Start the encounter.
3. See the results.

**Revision**: 5586

*Notes*:

1. For Health i will need to check them by taking some ss from server and then report them if need with proper info.
2. Any bug you find or know make a comment and will check and add it if it is true.

**Comment by Jenovas** [comment]
> 1. Pets cannot reach and attack the boss.(known by devs and is mmaps bug)

I won't be able to fix this. The workaround for now is click "Follow" for the pet to run to you and later click attack and the pet will attack. Players tested this solution and say it works.

> 3. It does not matter at which direction/angle you "hit" the Void of the Unmaking and it always follow a specific path.

This is not true. The Void should move almost on a straight line back and forth no matter what angle it was hit from.

> 4. The Void of the Unmaking should probably have less inner cooldown(1.5sec) to again be able to collide with a player.

I checked this and from my tests the cooldown should be around 4 - 5 secs. On the server the value is set to 5 secs.

**Comment by Aival94** [comment]
Yeah was mostly meaning about the almost same path..on server it follow same path exactly as far as i saw.(removed some fixed bugs and the seconds you said)

**Comment by Aival94** [comment]
Implementing totems damage during black phase and updating rev.
